### DISCUSSION TITLE
tag missing problem

### DISCUSSION THREAD

**Discussion by CChengjie** [vsch/flexmark-java#527] [Q&A] [answered]
I'm doing a research on tags .
We found that versions like **0.61.34** was released in Maven, but there was no corresponding tag in github, Can you help me with this?
1）What is the reason for tag's absence?

- a、This version code has problems, such as the loophole, compatibility and interface issues, etc.
- b、This version has functional problems, such as unreasonable function planning, basic function cannot be used
- c、Forgot to tag
- d、Don't care or is too trouble
- e、Don't care or is too trouble
- f、others，_

2） To solve this problem, we propose a method that can find the true commit of version to help developers quickly locate the problem code and fix the bug. Here are the possible true commits given by our method. Can you confirm whether or not the actual commits are included, and if not, which ones?

**['9b83b2e4122bf1e91fadcff825cd399f6e7549cd']**

♥ This question is very important to our research. Thank you for your time!

**Comment by vsch** [comment, marked as answer]
@CChengjie, I forgot to tag it. ['9b83b2e4122bf1e91fadcff825cd399f6e7549cd'] is the right commit. I added the tag and also 0.61.36, 0.62.4
